File: OCH_Tao/ABC/A/168.js
const hon = ["2", "4", "5", "7", "9"];
const pon = ["0", "1", "6", "8"];
const N = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0];
if (hon.includes(N[N.length - 1])) {
  console.log("hon");
} else if (pon.includes(N[N.length - 1])) {
  console.log("pon");
} else {
  console.log("bon");
}

File: OCH_Tao/ABC/A/168.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
  string N;
  cin >> N;
  char S = N.back();
  string hon = "24579";
  string pon = "0168";
  if (hon.find(S) != string::npos)
  {
    cout << "hon" << endl;
  }
  else if (pon.find(S) != string::npos)
  {
    cout << "pon" << endl;
  }
  else
  {
    cout << "bon" << endl;
  }
}